ios/event/RNSHeaderHeightChangeEvent.mm

```diff
@@ -0,0 +1,44 @@
+#import "RNSHeaderHeightChangeEvent.h"
```

Comment:
Why is the folder named `event` not `events` as on android?

Reply:
That's a good point! Changed in https://github.com/software-mansion/react-native-screens/pull/1802/commits/bb80aef75672c8f4aa9d13b956fe49e417be269d.